tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend_test_ci-pipeline.yml

```diff
@@ -0,0 +1,32 @@
+trigger: none
```

Comment:
nit: can we name this file "orttraining-linux-gpu-frontend-test-ci-pipeline.yml"? s/_/-/g